rllib/offline/estimators/doubly_robust.py

```diff
@@ -1,76 +1,109 @@
-from ray.rllib.offline.estimators.off_policy_estimator import OffPolicyEstimate
-from ray.rllib.offline.estimators.direct_method import DirectMethod, train_test_split
+import logging
+from typing import Dict, Any
+from ray.rllib.policy import Policy
+from ray.rllib.policy.sample_batch import MultiAgentBatch, DEFAULT_POLICY_ID
 from ray.rllib.utils.annotations import DeveloperAPI, override
+from ray.rllib.utils.framework import try_import_torch
 from ray.rllib.utils.typing import SampleBatchType
-from ray.rllib.policy.sample_batch import SampleBatch
-from ray.rllib.utils.numpy import convert_to_numpy
 import numpy as np
+from ray.rllib.utils.numpy import convert_to_numpy
+from ray.rllib.utils.policy import compute_log_likelihoods_from_input_dict
+
+from ray.rllib.offline.estimators.off_policy_estimator import OffPolicyEstimator
+
+torch, nn = try_import_torch()
+
+logger = logging.getLogger()
 
 
 @DeveloperAPI
-class DoublyRobust(DirectMethod):
-    """The Doubly Robust (DR) estimator.
+class DoublyRobust(OffPolicyEstimator):
+    """The Doubly Robust estimator with a trainable Q-model.
```

Comment:
**todo**: Same problem. The documentation is too abstract :)

Reply:
This file has the same nits as the comments in `direct_method.py` and probably all other files 😶‍🌫️

Reply:
Done